napari_aicssegmentation/view/workflow_steps_view.py

```diff
@@ -116,15 +125,50 @@ def _add_workflow_steps(self, category: WorkflowStepCategory):
 
         self.layout.addSpacing(10)
 
+    def _add_bottom_buttons(self):
+        layout = QHBoxLayout()
+        layout.setSpacing(5)
+
+        btn_close_workflow = QPushButton("Close workflow")
+        btn_close_workflow.setFixedWidth(120)
+        btn_close_workflow.clicked.connect(self._btn_close_clicked)
+
+        btn_run_all = QPushButton("Run all")
+        btn_run_all.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Preferred)
+        btn_run_all.clicked.connect(self._btn_run_all_clicked)
+
+        layout.addWidget(btn_close_workflow)
+        layout.addWidget(btn_run_all)
+
+        self.layout.addLayout(layout)
+
     #####################################################################
     # Event handlers
     #####################################################################
 
-    def _btn_back_clicked(self, checked: bool):
-        self._controller.navigate_back()
-
     def _btn_info_clicked(self, checked: bool):
-        self.diagram = QLabel()
         diagram_path = str(Directories.get_assets_dir() / "workflow_diagrams/sec61b_1.png")
         self.diagram.setPixmap(QPixmap(diagram_path))
         self.diagram.show()
+
+    def _btn_close_clicked(self, checked: bool):
+        prompt = (
+            "<span>You are closing an in-progress Allen Cell & Structure Segmenter plugin workflow to return "
+            "to the Workflow Selection screen.&nbsp;Your progress in this workflow will be lost.</span>"
+        )
+
+        self.confirmation_modal.setModal(True)
```

Comment:
Can we name this modal more explicitly? There will be other modals / message boxes eventually on this page, so `confirmation_modal` is too generic. Something like `modal_close_workflow` so we know what we're talking about.